docs-client/src/containers/App/index.tsx

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import AppBar from '@material-ui/core/AppBar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Drawer from '@material-ui/core/Drawer';
+import Hidden from '@material-ui/core/Hidden';
+import IconButton from '@material-ui/core/IconButton';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import {
+  createStyles,
+  Theme,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core/styles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import MenuIcon from '@material-ui/icons/Menu';
+import React from 'react';
+import { hot } from 'react-hot-loader';
```

Comment:
Yeah - the module name is on the right side. It is almost always all lowercase, but if it only exports a single class, sometimes it is uppercase (the case for the material ui exports). There is some history behind material ui exporting a class per module because webpack used to fail at tree shaking them when importing from the top-level. I feel this is probably not the case anymore, but they still document their imports in this form. In general it is considered bad practice for a library to not export everything from the top level though.

I stick to all lowercase all the time for modules for simplicity.

The left side is what is imported. A function is all lowercase while a class is camelcase.

Hope that answers the question :)

Reply:
Ah! Didn't know that. Thank you for letting me know. :)